std/variant.d

```diff
@@ -167,7 +184,7 @@ private:
     // state
     ptrdiff_t function(OpID selector, ubyte[size]* store, void* data) fptr
         = &handler!(void);
-    union
+    align(maxAlignment!(AllowedTypes)) union
     {
         ubyte[size] store;
```

Comment:
If you move the `align()` down here, you won't change the alignment for `AllowedTypes = { int[2], short[4] }` for 64-bit targets (so that it would still be 8-bytes pointer-aligned due to the pointers array). It doesn't really matter, as the function pointer field makes sure the whole `VariantN` struct is still at least pointer-aligned and that the union's offset is 8; it'd just be good practice.

Reply:
Don't really want to think about these alignment issues, just moved it down like you said: works on Android/x64.